src/main/java/com/sda/zdjavapol107/statki/MessagesPrinter.java

```diff
@@ -0,0 +1,90 @@
+package com.sda.zdjavapol107.statki;
+
+public class MessagesPrinter {
+
+    public void printStartingMenuLabel(){
```

Comment:
ok to da sie jeszcze troche bardziej zautomatyzowac :) 
zrobcie tak zeby stworzyc funkcje prywatna: 
```java
private void printHeader(String text){
  printLineOfSeparator();
  pringHeader(text);
  printLineOfSeparator();
}
```
BTW mozecie to zmienic na printHeaderSparator :) 
a funkcje printHeader zaimplementujcie sami. ma sama centrowac text i dodawac po bokach ---- tak zeby bylo rowno :D